src/includes/init.php

```diff
@@ -36,6 +38,13 @@ function redirect($url, $flash = null)
     exit();
 }
 
+function error($title, $body)
+{
+    $_SESSION['errtitle'] = $title ?? "fatal error";
+    $_SESSION['errcode'] = $body ?? "fatal error";
+    redirect("/includes/errorPage/errorPage.php");
```

Comment:
Consider passing `['errtitle' => $title, 'errcode' => $body]` to `redirect` function's $flash parameter and using `getFlash` in errorPage.php to fetch it. I'm thinking we can also pass an HTTP status code here and on to errorPage.php.

Reply:
It may also make more sense to put 'fatal error' as a default value right inside the function signature. Like `function error($title = 'fatal error', $body = 'fatal error') { ... }`.

Reply:
ok, I moved those over from null coalesce. you're right on it being way better